docs/system/update.md

```diff
@@ -0,0 +1,146 @@
+---
+title: System Update
+tags:
+  - HMX
+  - MX-4
+  - CT
+  - MX-V
+  - T30
+  - T30 FR
+  - C61
+  - Updating
+---
+
+## Overview
+
+Updating the software in Host Mobility hardware falls into two main categories:
+
+1. Installation or upgrade of specific software.
+2. System installation or re-installation.
+ 
+##  Installation or upgrade of specific software.
+
+###  Opkg package manager
+
+`opkg` is used  to install extra packages that are not installed with the original image.
+
+*To use it, first update package lists*
+```bash
+opkg update
+opkg list
+```
+
+*Example: install rsync*
+```bash
+opkg install rsync
+```
+
+### Editing package feeds
+
+The files in the folder `/etc/opkg/*` define where to get extra software. You can edit to point to a customer server.
+**NOTE: Make sure the package repository that you add is compatible and secure to use**
+
+## System installation or re-installation
+
+### System image
+
+Host Mobility hardware is able to update itself with two methods using files created by the build system:
+
+1. Put files on a USB memory and plug it into the hardware and press `reset-button`. *On the HMX, the USB-upgrade button must be held down during reset as well*
+2. Copy the files over the network to /boot directory and set the `firmware_update` u-boot environment variable to `true`
```

Comment:
this is only true for some units of mx4. C61 does no longer need that and not hmx. this need to be clearified. or set a refrence to mx-4 chapter below..